src/app/shared/chips/models/chpis.model.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { Chips } from './chips.model';
```

Comment:
This Spec file has a misspelling in the file name `chpis.model.spec.ts`

Reply:
fixed